bin/rails

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
Hi @chao-xian did you do the update via `rake app:update`? It might be a more safe way to do it.